realm/realm-library/build.gradle

```diff
@@ -257,35 +257,36 @@ def betaTag = 'Beta:a:<div style="border-style:solid;border-width:2px">This soft
         'considered at production quality, and should be used with care.</div>'
 
 task javadoc(type: Javadoc) {
-// FIXME: Disable JavaDoc until API Stabilizes a bit more
-//    source android.sourceSets.objectServer.java.srcDirs
-//    source android.sourceSets.main.java.srcDirs
-//    source "../../realm-annotations/src/main/java"
-//    classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
-//    options {
-//        title = "Realm ${project.version}"
-//        memberLevel = JavadocMemberLevel.PUBLIC
-//        docEncoding = 'UTF-8'
-//        encoding = 'UTF-8'
-//        charSet = 'UTF-8'
-//        locale = 'en_US'
-//        overview = 'src/overview.html'
-//
-//        links "https://docs.oracle.com/javase/7/docs/api/"
-//        links "http://reactivex.io/RxJava/javadoc/"
-//        linksOffline "https://developer.android.com/reference/", "${project.android.sdkDirectory}/docs/reference"
-//
-//        tags = [betaTag]
-//    }
-//    exclude '**/internal/**'
-//    exclude '**/BuildConfig.java'
-//    exclude '**/R.java'
-//    doLast {
-//        copy {
-//            from "src/realm-java-overview.png"
-//            into "$buildDir/docs/javadoc"
-//        }
-//    }
+    source android.sourceSets.objectServer.java.srcDirs
```

Comment:
Mostly rooted in history where we only have one link pointing to API Docs. It would probably make sense have two different links, but so far it hasn't been prioritized and should be coordinated with docs.